extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -1064,6 +1064,68 @@ describes.realWin('amp-ad-network-doubleclick-impl', realWinConfig, env => {
           ['https://partner.googleadservices.com', SAFEFRAME_ORIGIN]);
     });
   });
+
+  describe('Troubleshoot', () => {
```

Comment:
More descriptive name?

Reply:
Done. Note that "Troubleshoot" is a feature name.